src/app/cli/src/tests/full_test.ml

```diff
@@ -83,7 +83,8 @@ let run_test () : unit Deferred.t =
   setup_time_offsets () ;
   print_heartbeat logger |> don't_wait_for ;
   Parallel.init_master () ;
-  File_system.with_temp_dir "full_test_config" ~f:(fun temp_conf_dir ->
+  File_system.with_temp_dir (Filename.temp_dir_name ^ "/full_test_config")
```

Comment:
Prefer the `^/` operator for `Core`, it will ensure that the `/` is added between the paths if needed. Then you can just do `Filename.temp_dir_name ^/ "full_test_config"`